content/blog/2021-07-05-LFX-Blog-Nupur.md

```diff
@@ -0,0 +1,131 @@
++++
+date = "2021-07-05T11:04:54+00:00"
+title = "LFX spring Mentorship experience report -Nupur"
```

Comment:
A title that mentions the topic would be great, that way someone looking over the posts can know that this is about syslog